_devel/ci/build-in-container-clang.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+docker run \
+    --mount type=bind,src=/mnt/f/repo-matterfi/opentxs,dst=/home/src,readonly \
+    --mount type=bind,src=/home/pgawron/opentxs/output,dst=/home/output \
+    -i --entrypoint /usr/bin/build-opentxs-clang.sh \
+    polishcode/matterfi-ci-fedora:35-1 nopch
```

Comment:
Can we add an empty new line at the end of .sh scripts? Github is complaining about it. Same for files:
 - _devel/ci/build-in-container-gcc.sh
 - _devel/ci/run-container.sh
 - _devel/ci/test-in-container.sh
 - _devel/flatpak/run-container.sh

and also complains about this one:
_devel/flatpak/README

Reply:
fixed in second commit